src/coreComponents/fileIO/vtk/VTKPolyDataWriterInterface.cpp

```diff
@@ -114,10 +114,11 @@ std::pair< vtkSmartPointer< vtkPoints >, vtkSmartPointer< vtkCellArray > >VTKPol
                                                                                                                NodeManager const & nodeManager ) const
 {
   vtkSmartPointer< vtkPoints > points = vtkPoints::New();
-  points->SetNumberOfPoints( esr.size() + 1 );
+  localIndex const numPoints = esr.size() > 0 ? esr.size() + 1 : 0;
```

Comment:
It's `0` when you have nothing and it's `esr.size() + 1` because the number of points is equal of the number of elements (which are segments: 2 nodes) + 1? Is that it?

Maybe a small in-line comment?

Reply:
Thanks for looking at the PR. Yes, that is it. I added some comments at the top of this function to clarify the structure of the `WellElementSubRegion`, and why we need to set `numPoints` to zero when `esr.size()` is zero.